### PR TITLE
Update locale messages

### PR DESCRIPTION
**Pull request by Monkatraz**
This migrates/updates all of the new frontend's locale messages to the `/locales` system we have. I left the old Jetstream `web/resourceslang/en/...` files alone, but eventually they'll get deleted once we remove Jetstream.

### CODE REVIEW

File: web/app/Services/License/License.php [hunk 1]
```diff
@@ -37,16 +37,9 @@ private function buildHtml(): string
...
-            $prefix = __(
-                'Unless stated otherwise, the content of this page is licensed under',
-            );
-        } else {
-            $prefix = __('The content of this page is licensed under');
-        }
+        $prefix = $this->unlessClause ? 'Unless stated otherwise, the' : 'The';
 
-        return "$prefix $link.";
+        return "$prefix content of this page is licensed under $link.";
```
Comment (emmiegit) [inline web/app/Services/License/License.php:42]: Wait, why isn't this using the localization function anymore?

**Review by emmiegit — APPROVED** [review]

File: web/app/Services/License/License.php (hunk 1 above)
Comment (Monkatraz) [reply web/app/Services/License/License.php:42]: The new frontend doesn't use this at all, and I didn't want to add a new localization key for legacy content

Comment (emmiegit) [reply web/app/Services/License/License.php:42]: I see

**Comment by Monkatraz** [comment]
thanks @ammongit

**Pull request merged**